packages/lockfile-lint/src/validators/index.js

```diff
@@ -59,3 +81,35 @@ function ValidateHttpsManager ({path, type, validatorValues, validatorOptions})
 
   return validator.validate()
 }
+
+function ValidateUrlManager ({path, type, validatorValues, validatorOptions}) {
```

Comment:
Hmm, so this is needed, but only in the case that specific urls were specific `-u` and no hostnames, otherwise we don't need to trigger this one. In the case that we do need to trigger this only on specific URLs then the whole work with `ValidateHost` below is then needless.

Reply:
@bolatovumar ok so last comment before we merge - where do we stand on this one? does my comment make sense.

I understand that you've put it here because the validators are being run regardless to one another so maybe we can update the code for `runValidators()` to not run the url validator manager, if the host validator manager is run, and if so, then skip it. I guess it changes things a bit so I'm happy to take a look at refactoring this after landing this PR.

Reply:
@lirantal yeah, your comment makes sense. I will look into it.

Reply:
Ok. I'll get this merged later tonight or tomorrow and we can pick up the remaining work after this one gets in.

Reply:
@lirantal ok, I added an additional check to skip running the `ValidateUrlManager` if both the `--allowed-urls` and `--allowed-host` are used.